src/rssitem.cpp

```diff
@@ -176,13 +157,13 @@ bool RssItem::has_attribute(const std::string& attribname)
 std::string RssItem::get_attribute(const std::string& attribname)
```

Comment:
> Somehow this job was pushed into getters in RssItem and RssFeed. It doesn't belong there.

Nitpick: This still converts between locales in a getter of `RssItem`.
Would it be feasable to move the use of `utils::utf8_to_locale()` to the callers of `get_attribute()`?

Reply:
It's not feasible: `get_attribute` is an overloaded member function of `Matchable` abstract base class, so the caller has no idea if they need to convert the result or not.

It's debatable if the conversions are needed here at all, though. This method is called when evaluating filter expressions, so e.g. a title can be compared with a user-provided string. However, as I stated elsewhere, all strings are in UTF-8 internally, so the user-provided string should be in UTF-8 as well. Then we don't need any conversions here.

In this PR, I always followed existing code if I had any doubts. We didn't get any bug reports about this (yet), so presumably it works (or nobody uses it in a way that exposes the bug). So I'll keep converting here, just to preserve existing behaviour. Also, all these conversions will be straightened out as the code moves to Rust, which enforces the UTF-8 convention way stronger than we do.

Reply:
Sounds good, thanks!